MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1148 @@
+---
+layout: post
+title: Editors in .NET MAUI DataForm(SfDataForm) control | Syncfusion
+description: Learn about the supported editors in Syncfusion .NET MAUI DataForm (SfDataForm) control in mobile and desktop applications from a single shared codebase.
+platform: maui
+control: SfDataForm
+documentation: UG
+---
+
+# Editors in .NET MAUI DataForm (SfDataForm)
+
+The data form supports several built-in editors as follows:
+
+<table>
+<tr>
+<th>Editor name</th>
+<th>Supported Data Type/Attribute</th>
+<th>Input control loaded</th>
+</tr>
+<tr>
+<td>
+Text
+</td>
+<td>
+The string type property and any other type apart from the below specified cases.
+</td>
+<td>
+{{'[Entry](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/entry?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+MultilineText
+</td>
+<td>
+The string type property with multi line text.
```

Comment:
The string type property with [DataType(DataType.MultilineText)] attribute.

Reply:
multilinetext removed